src/dpsvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/dpsvalidationservice/GetValidOpenDFCMCase.java

```diff
@@ -20,8 +24,13 @@
             @ApiResponse(code = 200, message = "Successful operation")})
     public String getValidOpenDFCMCase(@ApiParam(value = "driversLicense", required = true) @RequestParam(value = "driversLicense", required = true) String driversLicense, @ApiParam(value = "surcode", required = true) @RequestParam(value = "surcode", required = true) String surcode) {
         /* TODO implement the logic as in the webmethods */
+        logger.fatal("getValidOpenDFCMCase is called");
```

Comment:
probably don't need this but I would change it to .info instead of fatal

Reply:
removed the if check and the fatal statements